dotnet-interactive.Tests/InMemoryJupyterKernelSpec.cs

```diff
@@ -25,12 +26,20 @@ public class InMemoryJupyterKernelSpec : IJupyterKernelSpec
             _error = error;
         }
 
+        public List<string> InstalledKernelSpecs
+        {
+            get => _kernelSpecs;
+        }
```

Comment:
Backing field and method names should match. Or you can use a get-only property instead of a backing field since you expose `List<string>` as the type.